doc/reports/spikes/metrics-what-we-want-spike.md

```diff
@@ -0,0 +1,18 @@
+# Monitoring Performance - What should we monitor?
+## Introduction
```

Comment:
🎨 Markdown linting - should have blank line before and after headings

Reply:
JG knows his MD